src/helpers/WalletLanguage.ts

```diff
@@ -0,0 +1,19 @@
+import * as english from './languages/english.json';
+import * as spanish from './languages/spanish.json';
```

Comment:
Please look into https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports, as we most likely don't want to load all the language files from the get-go; I'd say only English should be included by default.